src/detectTests.ts

```
@@ -0,0 +1,608 @@
+  let testId = generateUUID();
+  let ignore = false;
+
+  statements.forEach((statement) => {
+    let test: DetectedTest | undefined;
+    let statementContent = "";
+    let stepsCleanup = false;
+
+    switch (statement.type) {
+      case "testStart":
+        statementContent = statement[1] || statement[0];
+        const parsedTest = parseObject({ stringifiedObject: statementContent });
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)